src/Uno.UI/UI/Xaml/Controls/WebView/UnoWKWebView.iOSmacOS.cs

```diff
@@ -51,12 +59,23 @@ public UnoWKWebView() : base(CGRect.Empty, new WebKit.WKWebViewConfiguration())
 			OkString = ok;
 			CancelString = cancel;
 
+#if __IOS__
 			if (UIDevice.CurrentDevice.CheckSystemVersion(10, 3))
 			{
 				_errorMap.Add(NSUrlError.FileOutsideSafeArea, WebErrorStatus.UnexpectedServerError);
 			}
+#endif
 		}
 
+#if __MACOS__
+		public CGSize SizeThatFits(CGSize availableSize)
```

Comment:
This should be in the mixins or the layouter for the other controls that may not support this, I'll integrate that in another PR.

Reply:
After https://github.com/unoplatform/uno/pull/2926, it's appropriate to keep using this through the interface.